stargazer/stargazer.py

```diff
@@ -216,6 +217,10 @@ def append_notes(self, append):
         assert type(append) == bool, 'Please input True/False'
         self.notes_append = append
 
+    def show_stars(self, show):
+        assert type(show) == bool, 'Please input True/False'
+        self.show_stars = show
```

Comment:
Same comment as https://github.com/mwburke/stargazer/pull/55/files#r446540348 ... I don't think you tested this ;-)